src/functions-and-arrays.js

```diff
@@ -1,41 +1,106 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(a, b) {
+  return Math.max(a, b);
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(words) {
+  if (!words.length) {
+    return null;
+  } else if (words.length === 1) {
+    return words[0];
+  } else {
+    return words.sort((a, b ) => b.length - a.length)[0];
```

Comment:
Thats smart, i think you can also use Math.max([...words]), and you will get same output